toolkit/scss/forms/_summary.scss

```diff
@@ -179,7 +260,6 @@ a.summary-item-remove-link {
 
     span {
       display: block;
-      padding-left: 10px;
```

Comment:
Taking this `padding-left` out seems to have broken something in the example page:

![image](https://cloud.githubusercontent.com/assets/87140/14611269/39d8e82a-0589-11e6-8a03-7703bd48cead.png)